### PR TITLE
Enhance Documentation with README.md for Builder and Robot Docker Images

### PR DESCRIPTION
**Pull request by ayushishu**
This README.md documentation will help users better understand and utilize our Builder and Robot Docker images.

### CODE REVIEW

File: Builder/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+$ cd distribution
+```
+4) Run the Maven (MVN) build:
+```
+mvn clean install
+```
+This will initiate the build process for the OpenDaylight project. Make sure you replace the example Git URL (https://git.opendaylight.org/gerrit/integration/distribution) with the URL of the specific OpenDaylight repository you intend to build.
+
+By following these steps, you can use the provided Docker image to build OpenDaylight projects with a consistent development environment.
+
```
Comment (askb) [inline Builder/README.md:44]: PS remove the extra blank lines

File: Builder/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# OpenDaylight Builder Docker Image
+
+This Docker image is designed for building OpenDaylight projects. It provides a pre-configured environment for Maven (MVN) builds on both Ubuntu and CentOS bases.
```
Comment (askb) [inline Builder/README.md:3]: PS mention if you'll be building the project itself or just running the tests?

**Comment by ayushishu** [comment]
Update README.md for Builder

File: Robot/dockerfile [hunk 3]
```diff
@@ -6,14 +6,20 @@ RUN cd /etc/yum.repos.d/ \
...
+
 # Update the system and install essential dependencies
 RUN yum update -y \
     && yum install -y epel-release \
     && yum install -y git wget \
     && yum install -y openssl-devel bzip2-devel libffi-devel \
     && yum groupinstall "Development Tools" -y
 
-# Download, compile, and install Python 3.10.2
+# Download, compile,a and install Python 3.10.2
```
Comment (askb) [inline Robot/dockerfile:22]: typo remove 'a '

**Review by askb — CHANGES_REQUESTED** [review]

**Comment by ayushishu** [comment]
Remove the typo mistake

**Review by askb — APPROVED** [review]
LGTM

**Pull request merged**